qe-on-duty-reporter/main.py

```
@@ -0,0 +1,289 @@
+        config.reports_dir = os.path.join(args.output_dir, 'reports')
+
+    # Parse date and time
+    now = datetime.now()
+    report_date = args.date if args.date else now.strftime('%Y-%m-%d')
+    report_time = args.time if args.time else now.strftime('%H%M')
+    timestamp = f"{report_date}T{report_time[:2]}:{report_time[2:]}:00Z"
+
+    # Initialize GitHub CLI client
+    try:
+        gh_client = GHClient(config.gh_cli_path, config.max_retries)
+    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)